### ISSUE TITLE
Municipalities in the border of regions

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [ipeaGIT/geobr#46]
It would be interesting if the function (or functions) implemented to solve #38, #39, and #45 allows the user to choose what to do with municipalities that are not fully within the region. Some possibilities:

1. Include all municipalities as long as they have some overlap with the region (maybe the default)
2. Remove the municipalities in the border
3. Cut the polygos of municipalities  in the border in order to guarantee that the returned area is the same of the region

**Comment by rafapereirabr** [comment]
Pedro, is there an offical shape file with the boundaries of those regions (Amazon, Semi-arid etc)? Because if there is, it would be better to implement a function that loads those boundaries, and then the user can decide how she will proceed with the intersection of municipalities

**Comment by pedro-andrade-inpe** [comment]
My only point here is that it is necessary to take care when someone asks for the municipalities that belong to some of these regions. Once the package returns them, the user might believe in the output and ignore that some municipalities have only partial overlap, which might have an important impact on the analysis she wants to perform. If the package has some options related to the border, it might help the user to figure out that this point is important or not.

**Comment by rafapereirabr** [comment]
Yes. You’re are absolutely correct. My initial idea was to return municipalities “in” those regions. However, your point made realize it’s better to return the shape file of each region itself, so we leave to the user to do the intersection between region and municipalities. Thanks for the heads up 

Rafael H.M. Pereira

**Comment by pedro-andrade-inpe** [comment]
Great. Maybe one example of computing such queries could be described in a vignette.

**Comment by rafapereirabr** [comment]
Issues #38 and #45 are now closed, we have included the `read_amazon` and the `read_biomes` functions. These functions return the boundaries of those regions

As for issue #39, we should be fixing it soon. The definition of the Brazilian semi-arid regions is based on a list of municipalities. So in this particular case the function will return the `sf` object of all the municipalities included in the list. I guess we can close this issue then. 

I agree we should create another vignette demonstrating how to download these data sets via **geobr**
